### DISCUSSION TITLE
SAT STANDARD SETTING

### DISCUSSION THREAD

**Discussion by Cmod777** [btcpayserver/btcpayserver#5024] [Ideas / Feature Requests]
When creating a shop on the "pos" plugin you can define a reference currency including SAT or any fiat currency.
It would be useful, in order to entice the use of a SAT standard, that it would be possible to define a fixed price in fiat (obscured) but displayed in satoshi already from the store and not only at check-out.
Currently the call for real time change is made only at check-out, if one could insert a call every 30 minutes (for example) it would be possible to display a dynamic price in SAT almost realistically, the real price would be defined at check-out as now but one would have a fairly reliable preview of the price in SAT already before (in the choice of the item).

**Comment by dennisreimann** [comment]
I still have this on my bucket list of things I want to get to. We could also offer a websocket connection to update the prices whenever the rate in BTCPay Server gets refreshed. Been asked about this feature multiple times and would like to implement it, just haven't found the time to do so yet.

**Reply by Cmod777** [reply]
I have no technical dev skills, simply it would be useful and nice to eliminate displaying a fiat price if you then get an SAT payment. It is a matter of principle

**Comment by dstrukt** [comment]
conceptual ACK

**Reply by nerdyrugbyguy** [reply]
Seems that ideally the vendor would select the currency that the price is based on, and the shopper would select the currency units that the price is displayed in.
